src/ocl/convolutionocl.cpp

```diff
@@ -402,6 +377,84 @@ static void ConvForwardCheckNumerics(const Handle& handle,
     }
 }
 
+void ConvolutionDescriptor::ValidateConvTensors(const ConvTensors& tensors) const
```

Comment:
[Recommendation] Let's avoid tautology in the new code.
```suggestion
void ConvolutionDescriptor::ValidateTensors(const ConvTensors& tensors) const
```
BTW maybe it is better to keep this as a freestanding function and pass group_count as a parameter?

Reply:
https://github.com/ROCmSoftwarePlatform/MIOpen/issues/2444 created to handle this in case we decide to merge the PR as is: @junliume

Reply:
Done in latest commit.